torch/_inductor/cudagraph_trees.py

```diff
@@ -1281,7 +1326,7 @@ def add_function(
         static_input_idxs,
```

Comment:
nit: I guess it would be nice if this method made it more clear that you were actually also running the function at the same time